umbraco-cloud/product-upgrades/version-specific-upgrades/migrate-from-8-to-latest.md

```diff
@@ -0,0 +1,127 @@
+---
+description: >-
+  This article will provide steps on how to migrate a Cloud project from Umbraco 8 to the latest version of Umbraco.
+---
+
+# Migrate from Umbraco 8 to the latest version
+
+Since the underlying framework going from Umbraco 8 to the latest version has changed, there is no direct upgrade path. That said, it is possible to re-use the database from your Umbraco 8 project on your new project in order to maintain the content.
+
+It is not possible to migrate the custom code as the underlying web framework has been updated from ASP.NET to ASP.NET Core. All templates and custom code will need to be re-implemented.
+
+You also need to make sure that the packages that you are using are available on the latest version.
+
+## Prerequisites
+
+* A Umbraco 8 Cloud project running **the latest version of Umbraco 8**.
+* A clean Cloud project running the latest version of Umbraco **at least 2 environments**.
+* A backup of your Umbraco 8 project database.
+
+{% hint style="info" %}
+We strongly recommend having at least **2 environments** on the new Umbraco project.
+
+Should something fail during the migration, the Development environment can be removed and re-added to start over on a clean slate.
+{% endhint %}
+
+## Step 1: Content Migration
+
+1. Create a backup of the database from your Umbraco 8 project using the [database backup guide](../../databases/backups.md).
+    * Alternatively you can clone the environment down and take a backup of the local Database.
+    * Make sure to **restore both content and media** from your Cloud environment.
+2. Import the database backup into SQL Server Management Studio.
+3. Clone down the **Development** environment from the new Cloud project.
+4. Test the project and make sure to log in to the backoffice.
```

Comment:
Can we add a Note here stating not to restore anything from the cloud dev environment while spinning the new project? I encountered an issue here. I did a restore by mistake even though it was a blank project and then step 6 onwards did not work as expected.

Reply:
I kept getting the infinite login screen. I don't know if it's related or not but when I tried it again without restoring the project it worked fine.

Reply:
Ah, alright!

We could add a note saying:
"There is no need to use the restore option as there are no content on the Cloud project. Use the "Skip, and go .." (whatever that buttons says 😅 ) instead."

Then we don't say anything about issues, but simply suggest that they go straight to the backoffice? 🤔

Reply:
Yup, that sounds good 💪